spec/spec_helper.rb

```diff
@@ -1,7 +1,11 @@
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 
+require 'combustion'
 require 'stitch_fix/log_weasel'
+
+Combustion.initialize! :active_support
```

Comment:
I haven't heard of this before.  How is this different from requiring active support directly?